src/app/workflow/workflow-file-editor/workflow-file-editor.component.ts

```diff
@@ -107,7 +107,14 @@ export class WorkflowFileEditorComponent extends FileEditing {
           this.workflowsService.getWorkflow(editedWorkflow.id).subscribe(
             (newlyGottenWorkflow: Workflow) => {
               this.workflowService.setWorkflow(newlyGottenWorkflow);
-              this.alertService.detailedSuccess();
+              const mostRecentId = newlyGottenWorkflow.workflowVersions.reduce(
+                (max, n) => Math.max(max, n.id),
+                newlyGottenWorkflow.workflowVersions[0].id
+              );
+              const updatedVersion = newlyGottenWorkflow.workflowVersions.find(v => v.id === mostRecentId);
```

Comment:
You can also do something like stream().max(Comparator.comparing(WorkflowVersion::getId)) to get the version with the max id. Not sure if it is any better.

Reply:
I think that's Java :) Is there a corresponding TypeScript way?

Reply:
:man_facepalming: Still adjusting back to timezone